src/app/services/user-authentication.service.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable } from '@angular/core';
+import { User } from '../models';
+import { HttpClient } from '@angular/common/http';
+import { map } from 'rxjs/operators';
+const herokuUrl = 'https://project-backend.herokuapp.com';
```

Comment:
great that you moved it to a constant, but maybe it should be in capital letters

Reply:
+